public/index.html

```diff
@@ -60,9 +79,35 @@
           password: $('#login-form-password').val(),
         }, function(res) {
           if (res.err) return error(res.err);
-          $('#message').html(messageTmpl.render({message: `Welcome ${res.data.name}!`}));
+          $('#account').html(accountTmpl.render({
+            data: res.data,
+            isLoggedIn: true
+          }));
+          $('#message').html(messageTmpl.render({
+            message: `Welcome ${res.data.name}!`,
+          }));
+          $("#update-form").on('submit', function(e) {
+            e.preventDefault();
+            socket.emit('cl_update', {
+              name: res.data.name,
+              newName: $('#update-form-name').val(),
+              newBio: $('#update-form-bio').val(),
+            }, renderAM('Account updated successfully!'));
+          });
         });
       });
+
+      renderAM = function(message) {
```

Comment:
Remove this high-order function since it is just used once.
Just write the function declaration inside the function call above.